blackbox/docs/sql/system.txt

```diff
@@ -898,6 +898,24 @@ The table schema is as follows:
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+Accessing the ``sys.shards`` table is subjected to the same privileges 
+constraints as the other tables. 
+Namely, in order to query it, the connected user needs to have the ``DQL`` 
+privilege on the ``sys.shards`` table, either directly or inherited from the 
+``SCHEMA`` or ``CLUSTER``. 
+For more information on privileges inheritance see :ref:`Hierarchical 
```

Comment:
"For" should be on previous line. and actually sorry to change my mind, but let's put this in parenthesis again